business_requirement_gap_analysis_task/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================================
+Business Requirement Gap Analysis Task
+======================================
+
+This module extends the functionality of Business Requirement to support gap
+analysis and to allow you to find gap analysis task easily and create it
```

Comment:
to find gap analysis task easily and create it automatically.
=> to create gap analysis task and easily find it